bin/docker/Dockerfile

```diff
@@ -12,6 +17,9 @@ COPY ./metabase.jar /app/
 COPY ./run_metabase.sh /app/
 RUN chmod 755 /app/run_metabase.sh
 
+# tidy up
+RUN rm -rf /tmp/* /var/cache/apk/*
```

Comment:
because we're still running `apk add` above to install bash, I think we should leave this here to tidy up